controllers/teams.js

```diff
@@ -365,17 +365,14 @@ router.post('/', async (req, res, next) => {
 		delete req.body[feature];
 	});
 
-	api(req)
```

Comment:
an await in front of the api call and the removal of the catch L375-377 should have the same efect than adding the try/catch. however the next middleware will be called when an error is thrown. because of this the try/catch should not be required and i would just add an await and remove the catch which swallows the error.
If there is an error thrown above within of this method, the error will be catched in the next middleware too.